Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -17,6 +17,51 @@
 nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
 LG = nxDSCLog.DSCLog
 
+# Paths
+CONFIG_PATH = '/etc/opt/microsoft/omsagent/conf/'
+SERVER_ADDRESS = '/var/opt/microsoft/omsagent/run/npmdagent.sock'
+DEST_FILE_NAME = 'npmd_agent_config.xml'
+PLUGIN_PATH = '/opt/microsoft/omsagent/plugin/'
+PLUGIN_CONF_PATH = '/etc/opt/microsoft/omsagent/conf/omsagent.d/'
+RESOURCE_MODULE_PATH = '/opt/microsoft/omsconfig/modules/nxOMSAgentNPMConfig/DSCResources/MSFT_nxOMSAgentNPMConfigResource/NPM/'
+DSC_RESOURCE_VERSION_PATH = '/opt/microsoft/omsconfig/modules/nxOMSAgentNPMConfig/VERSION'
+AGENT_RESOURCE_VERSION_PATH = '/var/opt/microsoft/omsagent/state/npm_version'
```

Comment:
Keep in mind that this path will eventually need to incorporate the workspace ID to be multi-homing-compatible. However, this work does not need to be included in this PR.